jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/actions/QueryGroupAction.kt

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs.insights.actions
+
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.project.DumbAware
+import com.intellij.openapi.project.Project
+import software.aws.toolkits.jetbrains.core.explorer.actions.SingleResourceNodeAction
+import software.aws.toolkits.jetbrains.services.cloudwatch.logs.CloudWatchLogsNode
+import software.aws.toolkits.jetbrains.services.cloudwatch.logs.insights.QueryEditorDialog
+import software.aws.toolkits.resources.message
+class QueryGroupAction : SingleResourceNodeAction<CloudWatchLogsNode>(message("cloudwatch.logs.query")), DumbAware {
+    override fun actionPerformed(selected: CloudWatchLogsNode, e: AnActionEvent) {
+        val project: Project = selected.nodeProject
```

Comment:
when the type is obvious, we don't need to add the type. Also, I think this could be made 1 line and it would be fine, like:
```kt
QueryEditorDialog(selected.nodeProject, selected.logGroupName).show()
```